kola/harness.go

```diff
@@ -280,6 +280,7 @@ func RunTests(pattern, pltfrm, outputDir string) error {
 	}
 
 	if !skipGetVersion {
+		fmt.Printf("Creating cluster to check semver...\n")
```

Comment:
Should probably use plog

Reply:
Thanks, made the change! With `plog.Printf` it now shows the message at log level INFO which is good.

Reply:
I would just `plog.Info` explicitly to make it clear what log level is intended.

Reply:
Done! Makes sense to use the explicit call instead.